scripts/_0_2_8_human_to_adm_comparison.py

```diff
@@ -29,7 +29,7 @@
     "DryRunEval-MJ5-eval": ['Probe 1', 'Probe 1-A.1', 'Probe 1-B.1', 'Probe 2', 'Response 2-A.1-B', 'Response 2-B.1-B', 'Response 2-B.1-B-gauze-u', 'Response 2-A.1-B-gauze-sp', 'Probe 2-A.1-A.1', 'Probe 2-B.1-A.1', 'Probe 2-A.1-B.1-A.1', 'Probe 2-B.1-B.1-A.1', 'Probe 3', 'Probe 4']
 }
 
-def compare_probes(mongoDB, EVAL_NUMBER=4):
+def main(mongoDB, EVAL_NUMBER=4):
```

Comment:
This will always use an `EVAL_NUMBER` of 4, no?

Reply:
This is the other script I was mentioning in my other comment

Reply:
@dereknop I haven't looked at this yet, but please make sure if you haven't that the weekly script is updated with these new names